content/authors/shaumoumita/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+# Display name
+title: Moumita Shau
+
+# Username (this should match your github username)
+authors:
+- shaumoumita
+
+# Role/position e.g. Graduate student
+role: Postdoctoral Fellow
+
+# Organization membership
+Organizations/Affiliations:
+organizations:
+- name: University of Regina
+  url: https://www.google.ca
+
+# A list of your interests, add as many as you want
+interests:
+- Groups and representation Theory
+- Linear Algebra
+- Data Science
+- # Your educational history, add as many as you want
+education:
+  courses:
+  - course: PhD - Mathematics
+    institution: Univerisity of Regina
+  - course: MSc - Mathematics
+    institution: University of Calcutta
+
+# Social/Academic Networking, delete any items you don't want to use
+social:
+- icon: github
+  icon_pack: fab
+  link: https://github.com/pimsmath
+#- icon: twitter
+#  icon_pack: fab
+#  link: https://twitter.com/carlosshigoto
+#- icon: linkedin
+#  icon_pack: fab
+#  link: https://www.linkedin.com/in/carloscontrerasc/
+
+user_groups:
+- committee: members
+  role: Team 9 Serious Labs
+  tags:
```

Comment:
I think tags shouldn't be intented. Can you try changing this section to look like

```
user_groups:
- committee: members
  role: Team 9 Serious Labs

tags:
- '2021'
```

If you make that change and add a commit I think that should do the trick!